src/res/i18n/fr-FR/constants.json

```diff
@@ -1,4 +1,5 @@
 {
+    "localeName": "Français",
```

Comment:
i appreciate the concerns for this one

Reply:
Hehe, if only we had someone who understood french to translate this... 😉